Simplenote/Classes/SPOnboardingViewController.swift

```diff
@@ -221,7 +221,7 @@ private extension SPOnboardingViewController {
         hostingController.modalPresentationStyle = .formSheet
         hostingController.sheetPresentationController?.detents = [.medium()]
 
-        let presenter = presentedViewController ?? self
+        let presenter = navigationController?.visibleViewController ?? self
```

Comment:
Since now we may have multiple VCs in the hierarchy, we must access the UINavigationController